### ISSUE TITLE
Label issue in slide element

### ISSUE THREAD

**Issue by KingAndi** [node-red/node-red-dashboard#818]
Hallo!

I use a slider control with a custom label in the format "ABC {{msg.label}}". This works fine.
I also use `msg.enable` to enable and disable the slider, this also works fine.

But if `msg.label` is set to apsecific value (e.g. "Hallo World", so the label shows "ABC Hallo World") and after this  I send a `msg.enable` message (regardless enable is true or false) than the label looses the "Hallo World" (msg.label variable) and only displays the default "ABC". Of cause `msg.label` is NOT set in the `msg.enable` event!

Maybe someone can have a look at this issue?

Best regards, Andreas

**Comment by dceejay** [comment]
Yes - this is to be expected - If you need the label you need to send it. The value is calculated every time you send a msg.
